lib/verbalize/action.rb

```diff
@@ -17,20 +17,24 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
-        @inputs = required_keywords
+        @required_inputs = required_keywords
         @optional_inputs = optional
 
         class_eval Build.call(required_keywords, Array(optional))
       end
 
-      def inputs
-        @inputs || []
+      def required_inputs
+        @required_inputs || []
       end
 
       def optional_inputs
         @optional_inputs || []
       end
 
+      def inputs
+        required_inputs + optional_inputs
```

Comment:
This mashes everything together into a single array.  I'm not sure that is what we really want here.  Maybe a hash with keys for each?  Or an array with the same format as the argument list to the `input` method?

It's hard to know what the right implementation is here since we don't have a concrete use case